router/server.go

```diff
@@ -85,27 +84,31 @@ func main() {
 		}
 	}
 
-	// Read etcd addresses from ETCD
-	etcdAddrs := strings.Split(os.Getenv("ETCD"), ",")
-	if len(etcdAddrs) == 1 && etcdAddrs[0] == "" {
-		if externalIP := os.Getenv("EXTERNAL_IP"); externalIP != "" {
-			etcdAddrs = []string{fmt.Sprintf("http://%s:2379", externalIP)}
-		} else {
-			etcdAddrs = nil
-		}
+	postgres.Wait("")
+	db, err := postgres.Open("", "")
+	if err != nil {
+		shutdown.Fatal(err)
+	}
+	if err := migrateDB(db.DB); err != nil {
+		shutdown.Fatal(err)
 	}
-	etcdc := etcd.NewClient(etcdAddrs)
 
-	prefix := os.Getenv("ETCD_PREFIX")
-	if prefix == "" {
-		prefix = "/router"
+	pgxpool, err := pgx.NewConnPool(pgx.ConnPoolConfig{
+		ConnConfig: pgx.ConnConfig{
+			Database: os.Getenv("PGDATABASE"),
```

Comment:
This is insufficient to connect to postgres when running inside of flynn. I believe the deployer has an example of setting up a pgx connection pool.